updatesnap/SnapVersionModule/snap_version_module.py

```diff
@@ -0,0 +1,163 @@
+""" Extends ManageYAML and Snapcraft from snpamodule
```

Comment:
Why do you extend this instead of modifying the original classes?

Reply:
This was due to  limitation on style checks, where a single file is not permitted to exceed 1000 lines of code.

Reply:
In that case, I think that it would be better to separate manageYAML and Snapcraft classes in their own files...